packages/edit-widgets/src/components/widget-areas-block-editor-content/index.js

```diff
@@ -42,13 +39,15 @@ export default function WidgetAreasBlockEditorContent( {
 			<Notices />
 			<BlockTools>
 				<KeyboardShortcuts />
-				<BlockCanvas
```

Comment:
The block canvas also has a "block copy" hook and a "mouse move reset" one. Are these already used in the widgets editor somehow?

Reply:
> "block copy" hook

I missed that in #54207 the copy handler was moved to the `BlockCanvas` component. I added a copy handler and confirmed that copying the block works correctly.

>  "mouse move reset" one

Does this refer to the useMouseMoveTypingReset hook? If so, I looked into #54149 and it looks like it wasn't originally present in the Widget Editor 🤔